### PR TITLE
feat: add implementation of parseSection function

### LINKED ISSUE #16: Create implementation for parseSection function
Please, create implementation for parseSection function, which should get a section with some title from usage. For example, a section with title "Options:".

### PR DESCRIPTION
**Pull request by ifedchankau**
Add implementation of parseSection function.

Closes #16

### CODE REVIEW

File: src/handle.js [hunk 1]
```diff
@@ -13,8 +13,12 @@ const handle = (help) => {
 };
 
 const parseSection = (sectionName, help) => {
-    //TODO: create implementation
     let section = [];
+    let regularExp = new RegExp('[^\n]*' + sectionName + '[^\n]*\n?(?:[ \t].*?(?:\n|$))*');
```
Comment (xferra) [inline src/handle.js]: How about [string interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)?

File: src/handle.js [hunk 2]
```diff
@@ -13,8 +13,12 @@ const handle = (help) => {
...
+    let matches = help.match(regularExp);
+    matches.forEach(match => {
```
Comment (xferra) [inline src/handle.js]: Use [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map)?

**Review by romanrostislavovich — APPROVED** [review]
+1

**Pull request merged**